components/CreateInspection.js

```diff
@@ -0,0 +1,249 @@
+/* eslint-disable no-underscore-dangle */
+import React, { useState, useEffect } from 'react';
+import Router from 'next/router';
+import {
+  Button, Modal, ModalHeader, ModalBody, Label, Input, Form, FormGroup
+} from 'reactstrap';
+import axios from 'axios';
+import get from 'lodash/get';
+import isEmpty from 'lodash/isEmpty';
+import { url } from '../utils/config';
+import ErrorMessage from './ErrorMessage';
+
+const CreateInspection = () => {
+  const [isOpen, setIsOpen] = useState(false);
+  const [newInspection, setNewInspection] = useState({});
+  const [selectData, setSelectData] = useState({});
+  const [errors, setErrors] = useState([]);
+  const headers = {
+    'Content-Type': 'application/json'
+  };
+
+  const errorHandling = (err) => {
+    const message = get(err, 'response.data.errors.description.message', err.message);
+    setErrors([...errors, { message }]);
+  };
+
+  const dataHolder = {
+    client: [],
+    car: [],
+    employee: []
+  };
+
+  const dataTransformer = (res) => {
+    const { data } = res;
```

Comment:
As a fallback in case data is empty
```suggestion
    const { data = [] } = res;
```